backend/app/adapter/sqldb/user_short_link_integration_test.go

```diff
@@ -114,6 +114,128 @@ func TestListShortLinkSql_FindAliasesByUser(t *testing.T) {
 	}
 }
 
+func TestListShortLinkSql_IsAliasOwnedByUser(t *testing.T) {
+	now := mustParseTime(t, "2019-05-01T08:02:16Z")
+	user := entity.User{
```

Comment:
You can directly put the user into the test struct. Also fields LastSignedInAt, CreatedAt, UpdatedAt are redundant.

Reply:
I'm a bit confused as to the reasoning behind this. Writing the user in the test struct multiple times would just create duplicated code.

Reply:
As far as the time stamps go, they're set so that there is no nil pointer left which can lead to errors down the road 😄

Reply:
I was suggesting for easy readability but we can leave it as it is since it is readable also this way. Good point about timestamps, if we change functioning, those tests should not fail for nil pointers!

Reply:
@oatovar Making the test case easy to follow avoid distracting engineers reading the code away from coming up edge case.

Reply:
Let's remove the used fields and move user logic into the test case. We may have to modify the test case as the system evolve. Having user in the test case themselves allowing us to make changes without touching the rest.